models/staging/whatsapp-webhook/scr_webhook.yml

```diff
@@ -1,10 +1,10 @@
 version: 2
 
 sources:
-  - name: webhook
+  - name: whatsapp_webhook_prod
     description: dataset created by whatsapp webhooks on BigQuery
     database: cryptic-gate-211900
-    schema: 'whatsapp_webhook_prod'
+    schema: "whatsapp_webhook_prod_v2"
```

Comment:
Let's revert this. The `whatsapp_webhook_prod` dataset is in US region now

Reply:
Sure